app/scripts/controllers/UsernamePromptCtrl.js

```diff
@@ -0,0 +1,22 @@
+(function(){
+
+  function UsernamePromptCtrl($uibModalInstance, $cookieStore) {
+
+
+    this.submit = function() {
+      if(this.text) {
+
+        $cookieStore.put('blocChatCurrentUser', this.text);
```

Comment:
Yes, I just thought about that. The UsernamePromptCtrl is only available at the time of prompt.
So in this case it's okay to have the current user in the service, right?